src/Linux/R/Interpreters/Impl/RInterpreterInfo.cs

```diff
@@ -7,6 +7,9 @@
 using Microsoft.Common.Core.IO;
 using Microsoft.Common.Core.Services;
 using Microsoft.Common.Core.UI;
+using Microsoft.Common.Core.OS;
+using System.Collections.Generic;
+using System.Linq;
 
 namespace Microsoft.R.Interpreters {
     public sealed class RInterpreterInfo : IRInterpreterInfo {
```

Comment:
Ah, nevermind, I didn't notice that the other `RInterpreterInfo` is in the `Windows` namespace.